libs/aws/langchain_aws/retrievers/bedrock.py

```diff
@@ -123,11 +156,16 @@ def _filter_by_score_confidence(self, docs: List[Document]) -> List[Document]:
         return filtered_docs
 
     def _get_relevant_documents(
-        self, query: str, *, run_manager: CallbackManagerForRetrieverRun
+        self,
+        query: str,
+        *,
+        run_manager: CallbackManagerForRetrieverRun,
+        next_token: Optional[str] = None,
```

Comment:
Let's not add this. Users can use the kwargs to pass this if needed.

```suggestion
```

Reply:
Yes I removed it.